### ISSUE TITLE
Component: Wide Info Card

### ISSUE THREAD

**Issue by mtanco** [h2oai/wave#967]
**Humid Design System:** wide_info_card
**Card Name:** `wide_info_card`

![image](https://user-images.githubusercontent.com/4793561/129975936-07647651-1c5c-4dfd-a4e0-0e2d1393a4c0.png)

**Components:**
* Icon - proportionally-sized icon to the left that does not resize beyond a certain fixed width - required, uses a default like the `ui.header` object
* headline - optional
* title - Required
* subtitle - optional
* content - required

**Comment by mtanco** [comment]
Some but not all cards have a splash of color at the top. Should this be optional on every card?

**Comment by mtanco** [comment]
I see that we already have a `wide_info_card` but this seems to be more like a `wide_image_info_card`, no? @lo5 @mturoci

**Comment by mturoci** [comment]
> I see that we already have a wide_info_card but this seems to be more like a wide_image_info_card, no?

@mtanco current `wide_info_card` has both `image` and `icon` props so I think it makes sense to just update the designs of the existing card.

> Some but not all cards have a splash of color at the top. Should this be optional on every card?

IMO having it optional on every card can get misused by devs easily, resulting in ugly / weird apps. The question is what is the purpose of it. If it has a specific function, then I would go for including it on a certain elements only (for which it makes sense from a functional perspective). cc @sandruparo for more clarification on this. If it has no purpose, I suggest removing it. Open for discussion.

**Comment by shihan007** [comment]
@mturoci  @mtanco Yeah this line is used to keep the brand identity of yellow. Since we are moving with sdk colours now we can keep this as optional or remove this. Let me know what do you think?

**Comment by mturoci** [comment]
Thanks @shihan007, in that case I vote for removing it as people might start asking for the same in other cards, unconsciously misusing it and creating funky looking apps in the end. Wdyt @mtanco @lo5 ?

**Comment by iamabhishekmathur** [comment]
ping @mtanco @lo5

**Comment by mturoci** [comment]
Will go without the top color for now, can be added later if a proper usecase will be found.
